### PR TITLE
Feature/clean data

### PR DESCRIPTION
**Pull request by jasmyn2244**
### What (if any) features are you implementing?
- Data cleaning
  - format date
  - format money
### What (if anything) did you refactor?
- API calls in new file
### Were there any issues that arose?
- n/a
### Is there anything that you need from your teammate?
- n/a
### Refactor checklist
- [ ] Check variable and parameter names: are they descriptive and unique?
- [ ] Is the code DRY? SRP?
- [ ] Is it easy for others to read your code? (explanatory comments needed?)
- [ ] Is the syntax consistent? (e.g. use of semicolons, blank lines)
- [ ] Are commit messages formatted correctly?

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -14,55 +15,23 @@ class App extends React.Component {
...
   }
 
   componentDidMount = () => {
     getAllMovies()
-      .then(data => this.setState({ movies: data.movies }))
+      .then(cleanedData => this.setState({ movies: cleanedData }))
       .catch(error => this.setState({ error: error }))
   }
 
-  // cleanData = (data) => {
```
Comment (anewb87) [inline src/App.js:27]: Cleaning up juuuuunk!

File: src/App.js [hunk 2]
```diff
@@ -14,55 +15,23 @@ class App extends React.Component {
...
       isSelected: false,
       error: '',
-      errorMsg: ''
     };
   }
 
   componentDidMount = () => {
     getAllMovies()
-      .then(data => this.setState({ movies: data.movies }))
+      .then(cleanedData => this.setState({ movies: cleanedData }))
```
Comment (anewb87) [inline src/App.js:23]: Cleaned data!

File: src/Components/MovieDetails/MovieDetails.js [hunk 3]
```diff
@@ -1,37 +1,31 @@
...
 
-const MovieDetails = (props) => {
-
-  // formatMoney = (moneyValue) => {
-  //   return moneyValue.toLocaleString('en-US', {
-  //     style: 'currency',
-  //     currency: 'USD',
-  //   })
-  // }
+const MovieDetails = ({ selectedMovie, displayMain }) => {
```
Comment (anewb87) [inline src/Components/MovieDetails/MovieDetails.js:4]: destructuring

File: src/api-calls.js [hunk 4]
```diff
@@ -21,4 +18,61 @@ const checkResponseStatus = (response) => {
     }
 }
 
+const cleanAllMovies = ({ movies }) => {
```
Comment (anewb87) [inline src/api-calls.js:21]: Cleaning up our data!

**Review by anewb87 — APPROVED** [review]
Scrubba-dub-data

File: src/api-calls.js [hunk 5]
```diff
@@ -21,4 +18,61 @@ const checkResponseStatus = (response) => {
...
+            key: movie.id,
+            title: movie.title,
+            poster_path: movie.poster_path,
+            average_rating: movie.average_rating,
+        }
+    })
+
+    return cleanedMovies
+}
+
```
Comment (Gabby-Recny) [inline src/api-calls.js:34]: Great job cleaning the data! I want to see how we can make this function more SRP without having random methods in our API Calls.

**Pull request merged**